src/main/java/hello/AirBnbApplication.java

```diff
@@ -14,4 +18,26 @@ public static void main(String[] args) {
     }
 
 
+    @Bean
```

Comment:
Could move this into a security layer instead of having it inside this class - we can have a chat around it tomorrow or when you're free to see if we change the logic of this slightly